docs/docs/about/comparison-powershell.md

```diff
@@ -710,7 +710,7 @@ Get-FlowApprovalRequest|Microsoft.PowerApps.PowerShell|
 Get-FlowEnvironment|Microsoft.PowerApps.PowerShell|[flow environment list](../cmd/flow/environment/environment-list.md), [flow environment get](../cmd/flow/environment/environment-get.md)
 Get-FlowOwnerRole|Microsoft.PowerApps.PowerShell|
 Get-FlowRun|Microsoft.PowerApps.PowerShell|[flow run list](../cmd/flow/run/run-list.md), [flow run get](../cmd/flow/run/run-get.md)
-Get-PowerApp|Microsoft.PowerApps.PowerShell|
+Get-PowerApp|Microsoft.PowerApps.PowerShell|[pa app list](../cmd/app/app/app-list.md), [pa app get](../cmd/app/app/app-get.md)
```

Comment:
Paths to MD files are incorrect, should've been `../cmd/pa/app/app-list.md` and `../cmd/pa/app/app-get.md`

Reply:
Sorry for that! Guess I had thick fingers ;-)

Reply:
Very minor thing, don't worry about it. Just before I ran docs I was convinced it was a 💯 commit 💪